### PR TITLE
keyd: Add at 2.5.0

### PR DESCRIPTION
**Pull request by uni-dos**
**Summary**

Add keyd a key remapper for linux

Resolves getsolus/packages#3616

**Test Plan**

- Set up a config for vim movement
- open micro and make sure capslock+vim navigation works

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

**Comment by silkeh** [comment]
Thanks for the PR. However, keep in mind that we will not take a look until the package request has been accepted or rejected (see [the note in the documentation](https://help.getsol.us/docs/packaging/creating-a-new-package/)). I have set the PR to draft to indicate this.

Additionally, this was missing a `Resolves` part in the PR description as per the [commit message format for new packages](https://help.getsol.us/docs/packaging/creating-a-new-package#commit-message-format-for-new-packages). I've edited the PR to include this.

File: packages/k/keyd/package.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+license    : MIT
+component  : system.utils
+summary    : A key remapping daemon for linux.
+description: |
+    keyd provides a flexible system wide daemon which remaps keys using kernel level input primitives (evdev, uinput).
+build      : |
+    %make
+install    : |
+    %make_install PREFIX=/usr SOCKET_PATH=/run/keyd.socket
+    sed -e 's#@PREFIX@#/usr#' -e 's#multi-user.target#systemd-udev.service#' keyd.service.in > keyd.service
```
Comment (ReillyBrogan) [inline packages/k/keyd/package.yml:16]:
```suggestion
    sed -e "s#@PREFIX@#%PREFIX%#" -e 's#multi-user.target#systemd-udev.service#' keyd.service.in > keyd.service
```

File: packages/k/keyd/package.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    install -Dm00644 keyd.service -t $installdir/usr/lib/systemd/system/
+
+    # Start service by default
+    install -dm00755 $installdir/usr/lib/systemd/system/multi-user.target.wants
+    ln -sv ../keyd.service $installdir/usr/lib/systemd/system/multi-user.target.wants/keyd.service
+    rm -r $installdir/etc/
```
Comment (ReillyBrogan) [inline packages/k/keyd/package.yml:22]: We like to see what's happening.
```suggestion
    rm -rv $installdir/etc/
```

**Review by ReillyBrogan — CHANGES_REQUESTED** [review]
Also needs monitoring.yml

**Review by malfisya — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**